### DISCUSSION TITLE
Got rejected even with all the information filled and a official school certificate

### DISCUSSION THREAD

**Discussion by LunaHD24** [community/community#139961] [GitHub Education] [answered]
### Hi everyone,

Hey,
I've now applied for the second time. The first time was during the holidays where I could only provide a screenshot of my school homepage while being logged in (which can easily be faked so of course it got rejected).
Today I got an official school certificate saying that I visit the school ("Bescheinigung über Schulbesuch", the document is in german or "Certificate of school attendance" in english). It's dated, includes my full name, birth date, current grade, address, the school name and address, the date I started school there, a stamp of my school and it's also signed by our school principal. I've filled in my billing information, shared my location, selected the correct school, took the photo with OBS Virtual Cam. I used the 'Enrollment letter' option and 2FA is enabled.
I've attached a screenshot of the rejection reasons, the only thing I could maybe think about would be that I need to also provide the translated version (a friend of mine did it excactly like me and got accepted so I assumed I didn't need to).
Could anyone maybe help me on how I won't get rejected again? Thanks in advance!

**Comment by AidenJamesYt** [comment]
Could you provide a screenshot of the rejection reasons, if possible?

**Reply by LunaHD24** [reply]
Oh, sorry. I must've forgot to, here it is:
![Screenshot 2024-09-27 163045](https://github.com/user-attachments/assets/68849263-c10d-4fbc-b490-6ff65ae1e448)

**Comment by cgevans** [comment, marked as answer]
There appear to be numerous, arbitrary checks being done by an entirely automated process.  While I think there was manual review for cases the system couldn't automatically understand previously, it appears the process is now entirely automated.  It also appears to consider emojis as professional means of email communication.  

One check appears to be based on the *name* of the camera device.  I used an OBSBot (an external USB camera, not OBS Studio) to try to get higher resolution, and one of my rejection reasons was that I did not use my device camera, so it seems likely there is a substring check for "OBS" that results in rejection, or some other form of rejecting based off the device information, in order to prevent virtual devices .  Similarly, while options are provided to upload images, that appears to just result in an auto-rejection: in fact, numerous options that are supposedly given seem to lead to that.

The actual documents appear to just be being OCR'd and searched for specific text (full name, name of the university as known by Github, which may not be the official or full university name, a date, a title the process understands as faculty for faculty).  So it appears you will also be rejected if those exact strings aren't recognizable via whatever OCR the process is using, either because they are not there, are in another language, or are too blurry.  The document being more or less formal or official does not appear to matter, only those exact strings and their ability to be recognized.  In fact, more formal documents seem worse, eg, my employment contract with the official, long form name of the university appears to be rejected because Github has the abbreviated name, and likely because it has too much small text.  I eventually needed to add annotations to my paystub to include the university name in the correct form and language.

**Comment by davevad93** [comment]
Hi @LunaHD24 ,

As I said here #139950 , I suggest you to upload a photo of both the original and the English translated academic documentation with your full name, school's name and a current date in a `DD/MM/YYYY`, `MM/DD/YYYY` or `YYYY/MM/DD` format. The tips from @cgevans are also very helpful.

For more information, you can check the [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352).

**Comment by LunaHD24** [comment]
Thank you all! I will follow the tips and also translate it so I have better chances at getting accepted. If it still doesn't work I guess I will write their support :)

**Comment by scottyedog** [comment]
I have done everything that the teacher education requires and it still rejects it.  
Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Dec 17, 2024 for {my profile name}
(https://urldefense.com/v3/__https://github.com/scottyedog__;!!KGKeukY!0EvwZWWgPayqRhZQUrYgU_9rIdCyj4seJ4KUFVWWGCxqcKVy0Ny5D9hFTgaVR4kxeYzepyDkmSh19BdewEz9bw$).

Please secure your GitHub account with [two-factor authentication](https://urldefense.com/v3/__https://github.com/settings/security__;!!KGKeukY!0EvwZWWgPayqRhZQUrYgU_9rIdCyj4seJ4KUFVWWGCxqcKVy0Ny5D9hFTgaVR4kxeYzepyDkmSh19Bd2GkLe8Q$). You may need to log out and log back in to GitHub before reapplying.

Once you have resolved the issues with your sumbission, [visit us again and reapply](https://urldefense.com/v3/__https://education.github.com__;!!KGKeukY!0EvwZWWgPayqRhZQUrYgU_9rIdCyj4seJ4KUFVWWGCxqcKVy0Ny5D9hFTgaVR4kxeYzepyDkmSh19BdWgcWPIw$).
